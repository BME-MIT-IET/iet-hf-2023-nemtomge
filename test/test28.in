loadmap map.json  
start V1  
set Scientist V1 PL1
set Material ST1 aminoacid 10
move V1 ST1
touch V1  
0
move V1 PL1
give Active V1 Bear 1
move V1 ST1
touch V1 
0
exit
