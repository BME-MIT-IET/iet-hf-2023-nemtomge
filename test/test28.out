Map from file map.json loaded
The game has been started. Players are: V1   
Scientist V1 set on field PL1
Material aminoacid: 10 has been placed on field ST1
Scientist V1 moved to field ST1 
Choose one to pick: 
0 - cancel
1 - aminoacid(10)
Scientist V1 moved to field PL1
Scientist V1 has been given active agent Bear time: 1 
Scientist V1 moved to field ST1
Choose one to pick: 
0 - cancel
